lib/dashboard/monitor.js

```diff
@@ -368,7 +368,8 @@ class Dashboard {
     const self = this;
     self.logText.log('console> '.bold.green + cmd);
     self.console.executeCmd(cmd, function (result) {
-      self.logText.log(result);
+      if (result !== undefined) 
```

Comment:
undefined sometimes is a valid thing to print we shouldn't include it (the user can actually execute js in the console)

Reply:
ok, but the monitor automatically logs whatever is returned in the callback so what's the best way to address this? If 'undefined' return ''?